nomad/leader.go

```diff
@@ -532,8 +532,12 @@ func (s *Server) reapFailedEvaluations(stopCh chan struct{}) {
 			// due to the fairly large backoff.
 			followupEvalWait := s.config.EvalFailedFollowupBaselineDelay +
 				time.Duration(rand.Int63n(int64(s.config.EvalFailedFollowupDelayRange)))
+
+			// TODO: what is this followup eval and where to create timestamp?
 			followupEval := eval.CreateFailedFollowUpEval(followupEvalWait)
 			updateEval.NextEval = followupEval.ID
+			updateEval.CreateTime = time.Now().UTC().UnixNano()
```

Comment:
Remove this line, you are modifying an existing eval so it should not set CreateTime on `updateEval`.